Fenwick_Tree/cpp/fenwick.cpp

```diff
@@ -0,0 +1,45 @@
+#include <bits/stdc++.h>
+using namespace std;
+
+#define MAXN 100010
+int fenwick[MAXN], n;
+
+void update(int index, int val) {
+  for(int i = index; i <= n; i += index & (-index)) 
```

Comment:
Missing space before ( in for(  [whitespace/parens] [5]

*Origin: CPPLintBear, Section: `all.cpplint`.*